spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/FileSpec.scala

```diff
@@ -67,5 +69,30 @@ class FileSpec extends FlatSpec with Matchers {
 
     testModule should be(module)
   }
+  "download files according to URL" should "work properly" in {
+    val tmpFile = java.io.File.createTempFile("module", ".bigdl")
```

Comment:
why use a local temp file in this unit test? You should use some url start with http/https, right?

Reply:
Well, this is only because I am not sure weather 'Jenkins' will perform well when it involves Internet. But I have used the method to download a image from Internet on my own computer and it worked.